docs/src/main/sphinx/installation/cli.rst

```diff
@@ -13,28 +13,107 @@ Requirements
 The CLI requires a Java virtual machine available on the path.
 It can be used with Java version 8 and higher.
 
+The CLI uses the HTTP protocol and the :doc:`Trino client REST API
+</develop/client-protocol>` to communicate with the coordinator on the Trino
+cluster.
+
 Installation
 ------------
 
-Download :maven_download:`cli`, rename it to ``trino``,
-make it executable with ``chmod +x``, then run it:
+Download :maven_download:`cli`, rename it to ``trino``, make it executable with
+``chmod +x``, and run it to show the version of the CLI:
 
 .. code-block:: text
 
-    ./trino --server localhost:8080 --catalog hive --schema default
+    ./trino --version
+
+Run the CLI with ``--help`` or ``-h`` to see all available options.
+
+Running the CLI
+---------------
+
+The minimal command to start the CLI in interactive mode specifies the URL of
+the coordinator in the Trino cluster:
+
+.. code-block:: text
+
+    ./trino --server http://trino.example.com:8080
+
+Many other options are available to further configure how to start the CLI:
 
-Run the CLI with the ``--help`` option to see the available options.
+.. list-table::
+  :widths: 40, 60
+  :header-rows: 1
 
-The CLI uses the HTTP protocol and the
-:doc:`Trino client REST API </develop/client-protocol>` to communicate
-with Trino.
+  * - Option
+    - Description
+  * - ``--catalog``
+    - Sets the default catalog. You can change the default catalog and schema
+      with :doc:`/sql/use`.
+  * - ``--client-info``
+    - Add extra information about the client.
+  * - ``--client-request-timeout``
+    - Client request timeout (default: 2m).
+  * - ``--client-tags``
+    - Add extra tags information about the client, and the CLI user. The tags
+      can be used as input for resource groups. :doc:`/admin/resource-groups`.
+  * - ``--debug``
+    - Enable debug information for :ref:`cli-troubleshooting`.
+  * - ``--disable-compression``
+    - Disable compression of query results.
+  * - ``--editing-mode``
+    - Allows switching key bindings in the CLI.
```

Comment:
you need to add vi or emacs as valid choices here and also say what the default is